File: src/render/visualDebugTools/GlutDebugState.cpp
#include "render/visualDebugTools/GlutDebugState.h"

GlutDebugState GLOBAL_render_glutDebugState;

GlutDebugState::GlutDebugState():
        primarySelectedPatch(0),
        showSelectedPathOnly(true),
        angleAroundViewportU(0.0f),
        angleAroundViewportV(0.0f)
{
}

File: java/src/main/java/vsdk/toolkit/render/jogl/visualDebugTools/GlutDebugState.java
package vsdk.toolkit.render.jogl.visualDebugTools;

public class GlutDebugState {
    public int primarySelectedPatch;
    public int selectedSelectedPatch;
    public boolean showSelectedPathOnly;
    public float angleAroundViewportU;
    public float angleAroundViewportV;

    public GlutDebugState() {
        primarySelectedPatch = -1;
        selectedSelectedPatch = -1;
        showSelectedPathOnly = true;
        angleAroundViewportU = 0.0f;
        angleAroundViewportV = 0.0f;
    }
}
